contact.html

```diff
@@ -0,0 +1,245 @@
+                </svg>
+                <h2 class="contact-info__title">Email</h2>
+                <p class="contact-info__text">info.avitex@gmail.com</p>
+            </a>
+            <a href="tel:+16668888" class="contact-info__item">
+                <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M19.554 20.9008C21.178 24.2833 23.3918 27.4534 26.1955 30.2571C28.9992 33.0608 32.1693 35.2746 35.5518 36.8986C35.8427 37.0383 35.9882 37.1081 36.1723 37.1618C36.8264 37.3525 37.6297 37.2155 38.1837 36.8188C38.3396 36.7072 38.473 36.5738 38.7397 36.3071C39.5555 35.4914 39.9633 35.0835 40.3735 34.8168C41.9202 33.8112 43.9142 33.8112 45.461 34.8168C45.8711 35.0835 46.279 35.4914 47.0948 36.3071L47.5494 36.7618C48.7895 38.0018 49.4095 38.6219 49.7463 39.2877C50.4161 40.6121 50.4161 42.176 49.7463 43.5003C49.4095 44.1662 48.7895 44.7862 47.5494 46.0263L47.1816 46.3941C45.9458 47.6299 45.328 48.2477 44.4879 48.7197C43.5557 49.2433 42.1079 49.6198 41.0387 49.6166C40.0752 49.6138 39.4166 49.4269 38.0996 49.053C31.0217 47.0441 24.3428 43.2537 18.7709 37.6817C13.1989 32.1098 9.40848 25.4309 7.39955 18.353C7.02573 17.036 6.83882 16.3774 6.83596 15.4139C6.83278 14.3447 7.20929 12.8969 7.73293 11.9647C8.20485 11.1246 8.82274 10.5067 10.0585 9.27096L10.4263 8.90314C11.6664 7.66311 12.2864 7.0431 12.9523 6.70629C14.2766 6.03646 15.8405 6.03646 17.1648 6.70629C17.8307 7.04309 18.4508 7.66311 19.6908 8.90314L20.1455 9.35784C20.9612 10.1736 21.3691 10.5815 21.6358 10.9916C22.6414 12.5384 22.6414 14.5324 21.6358 16.0791C21.3691 16.4893 20.9612 16.8971 20.1455 17.7129C19.8788 17.9796 19.7454 18.113 19.6338 18.2689C19.2371 18.8229 19.1001 19.6262 19.2908 20.2803C19.3444 20.4644 19.4143 20.6099 19.554 20.9008Z" stroke="#F8F7FF" stroke-width="4" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+                <h2 class="contact-info__title">Phone</h2>
+                <p class="contact-info__text">+1 666 8888</p>
+            </a>
+        </section>
+
+        <section class="contact">
+            <h2 class="subtitle">Get in touch</h2>
+            <p class="contact__subtitle">Browse through the most freuently asked questions</p>
+
+            <form action="#" method="post" class="contact__form">
+            <div class="form-switcher">
```

Comment:
I know that this switcher from our lections, but I think in this case text also should be clickable. 
Please replace `<div class="form-switcher">` to the label `<label>`
And add cursor: pointer in css for label.

Reply:
done, added 2th label

Reply:
Unexpected solution!
I expect you made whole .form-switcher as label

<img width="676" alt="Screenshot 2024-12-16 at 01 33 03" src="https://github.com/user-attachments/assets/46ee099d-4f6b-4045-bc0e-4acda8ca0b51" />

Ort more complicated option: 
text it's label (for="") and transparent checkbox(position: absolute) covers custom switcher.